server/routes/overview/overviewController.test.ts

```diff
@@ -125,7 +125,7 @@ describe('overviewController', () => {
         counts: {
           inductionSessions: 1,
           reviewSessions: 1,
-          totalSessions: 2,
+          totalCompletedSessions: 2,
```

Comment:
As well as fixing the bug, I've renamed the terminology to be the total completed sessions
(because just having an Induction record does not mean the Induction is complete - you need at least 1 goal as well for it to be considered complete)